<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="16" hostname="SEZ-LAP-123" name="testcases.RAIS.SanityTestPack_Final" tests="17" failures="1" timestamp="08 Jan 2020 08:36:03 GMT" time="74.629" errors="0">
  <testcase name="Entity_CRUD_Operations" time="74.629" classname="testcases.RAIS.SanityTestPack_Final">
    <failure type="java.lang.AssertionError" message="expected [RAIS+] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [RAIS+] but found [null]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at testcases.RAIS.SanityTestPack_Final.Entity_CRUD_Operations(SanityTestPack_Final.java:1682)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- Entity_CRUD_Operations -->
  <testcase name="DataRole_CreateEditDeleteRole" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- DataRole_CreateEditDeleteRole -->
  <testcase name="LoginScreen_Default_Backup" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- LoginScreen_Default_Backup -->
  <testcase name="LoginScreen_Default_ExternalLogin" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- LoginScreen_Default_ExternalLogin -->
  <testcase name="DataRole_AddNewPermission_ValidationOld" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- DataRole_AddNewPermission_ValidationOld -->
  <testcase name="OLDSecurityRole_CreateEditDeleteRole" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- OLDSecurityRole_CreateEditDeleteRole -->
  <testcase name="LoginApplication" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- LoginApplication -->
  <testcase name="QATest" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- QATest -->
  <testcase name="UserProfile_Validation" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- UserProfile_Validation -->
  <testcase name="LoginScreen_Temp1" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- LoginScreen_Temp1 -->
  <testcase name="ReadEntityTable" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- ReadEntityTable -->
  <testcase name="SecurityRole_CRUD_Operations" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- SecurityRole_CRUD_Operations -->
  <testcase name="DataRole_AddNewPermission_Validation" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- DataRole_AddNewPermission_Validation -->
  <testcase name="DataRole_AddNewRestriction_Validation" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- DataRole_AddNewRestriction_Validation -->
  <testcase name="DataRole_Permission_Validation_AddEditDelete" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- DataRole_Permission_Validation_AddEditDelete -->
  <testcase name="FunctionalRole_CRUD_Operations" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- FunctionalRole_CRUD_Operations -->
  <testcase name="Forgot_PasswordFeature" classname="testcases.RAIS.SanityTestPack_Final">
    <skipped/>
  </testcase> <!-- Forgot_PasswordFeature -->
</testsuite> <!-- testcases.RAIS.SanityTestPack_Final -->
